File: Hello.java
class Hello {
	public static void main(String[] args) {
		System.out.println("Hello World");

		System.out.println("\nYour arguments:");
		for (int i = 0; i < args.length; ++i) {
			System.out.println(i + ".\t" + args[i]);
		}
	}
}

File: hello.c
#include <stdio.h>

int main(int argc, char *argv[])
{
	printf("Hello World\n");

	printf("Your arguments:\n");
	for (int i = 0; i < argc; ++i)
	{
		printf("1.\t%s\n", argv[i]);
	}

	return 0;
}
